shared/src/commonMain/kotlin/org/hyperskill/app/study_plan/widget/presentation/StudyPlanWidgetFeature.kt

```diff
@@ -93,7 +97,7 @@ object StudyPlanWidgetFeature {
     }
 
     internal sealed interface InternalAction : Action {
-        object FetchStudyPlan : InternalAction
+        data class FetchStudyPlan(val delay: Duration? = null, val tryNumber: Int = 1) : InternalAction
```

Comment:
Please add documentation for this action and its parameters

Reply:
Maybe we can find better names for the parameters?
1. `delay` -> `delayBeforeAttempt`
2. `tryNumber` -> `attemptNumber | currentAttempt | attemptCount`